frontend/src/bundles/common/helpers/date-converter/date-converter.ts

```diff
@@ -0,0 +1,8 @@
+import moment from 'moment';
+
+const dateConverter = (date: string): string => {
+    const convertedDate = new Date(date);
+    return moment(convertedDate).format('dddd, MMMM | h:mm A');
+};
```

Comment:
I don't think you should use it for formatting date. Since in react-multi-date-picker library that has DateObject type that can convert date in specified format. You can try to use it.

Reply:
Thanx, have not noticed that we using that library